sql/src/planner.rs

```diff
@@ -645,7 +645,10 @@ pub fn parse_for_option(value: Value) -> Result<Option<String>> {
         }
         Value::Boolean(v) => Some(v.to_string()),
         Value::Interval { value, .. } => {
-            return UnsupportedOption { value }.fail();
+            return UnsupportedOption {
+                value: format!("{}", value),
```

Comment:
Is there any method such as `to_string` defined on value?

Reply:
> Is there any method such as `to_string` defined on value?
It dosen't impl ToString trait ...

Reply:
It may have:

Expr implements `Display` https://docs.rs/sqlparser/latest/sqlparser/ast/enum.Expr.html#impl-Display-for-Expr, which means it's also `ToString` https://doc.rust-lang.org/std/string/trait.ToString.html

Reply:
> > Is there any method such as `to_string` defined on value?
> > It dosen't impl ToString trait ...

Thanks, I change it now.

Reply:
Done.